src/adhocracy_core/adhocracy_core/rest/__init__.py

```diff
@@ -1,4 +1,74 @@
 """Configure rest api packages."""
+from pyramid.view import view_config
+from adhocracy_core.authentication import validate_user_headers
+from adhocracy_core.caching import set_cache_header
+from adhocracy_core.rest.schemas import validate_request_data
+from adhocracy_core.rest.schemas import validate_visibility
+
+
+class api_view(view_config):  # noqa
+    """ A function\class\method decorator to configure a rest api :term:`view`.
```

Comment:
> I think we should use from functools import wraps otherwise the docstring of the wrapped method will disappear when using this decorator.

I have no strong opinion in this but but I would keep it that way to make it work like the normal view_config decorator.